SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -101,6 +115,26 @@ private static bool ExecuteJavaRunner(ILogger logger, string exeFileName, string
             return success;
         }
 
+        /// <summary>
+        /// Get the value of the SONAR_RUNNER_OPTS variable that controls the amount of memory available to the JDK so that the sonar-runner doesn't 
+        /// hit OutOfMemory exceptions. If no env variable with this name is defined at machine or user level then use the process one is used. 
+        /// Bar that, a default value is used. 
+        /// </summary>
+        /// <returns></returns>
+        private static string GetSonarOptsValue()
```

Comment:
The variable is called "SonarRunnerOpts" so this is what should appear in the method name. Shortening it to "SonarOpts" doesn't gain anything and introduces the possibility of confusion.